src/views/layouts/SidebarLayout.vue

```diff
@@ -78,6 +76,17 @@ export default {
             ]
         };
     },
-    methods: {}
+    mounted() {
+        const scrollElement = document.querySelector(".v-navigation-drawer__content");
+        scrollElement.addEventListener("scroll", this.handleScroll);
+    },
+    destroyed() {
+        // this.$el.removeEventListener("scroll", this.handleScroll);
+    },
+    methods: {
+        handleScroll() {
+            EventBus.get().emit(EventBus.NAVBAR_SCOLL_EVENT);
```

Comment:
This could probably do with a debonucer in future as it's going to get called _a lot_